lib/ApiRequestor.php

```diff
@@ -141,13 +141,16 @@ private static function _defaultHeaders($apiKey)
 
         $langVersion = phpversion();
         $uname = php_uname();
+        $curlVersion = curl_version();
```

Comment:
I ran some basic benchmarking, and it takes approximately 1.3 seconds to run 1 million calls to `curl_version` on my laptop. Given that ~all of our users will never send more than 100 req/s, I think we're fine.